src/DotNetStatus/Common/AnalyzerInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Fx.Portability
+{
+    public class AnalyzerInfo
+    {
+        public string AnalyzerName { get; set; }
+        public string Id { get; set; }
+        public ICollection<CompatibilityRangeAttribute> Attributes { get; set; }
+        public bool IsCompatibilityDiagnostic { get; set; }
+
+        public int IdNumber
+        {
+            get
+            {
+                var matches = Regex.Match(Id, @"(\d+)");
```

Comment:
This property is currently serialized and returned. Do you use it? If not, don't include it. I would like to change the service so that this property is not included. If you do use, switch it to a method like GetIdAsNumber()